### ISSUE TITLE
Fix missing alt attributes in dynamically loaded images

### ISSUE THREAD

**Issue by Pedrinhho** [Pedrinhho/gerenciaBanco#60]
Images loaded dynamically lack alt attributes, reducing accessibility. Add alt text support.
